tests/e2e/test_e2e_admin.py

```diff
@@ -1,14 +1,28 @@
 import re
 import subprocess
-from typing import Any
 
 import pytest
 
 from tests.e2e import Helper
 
 
+pytestmark = pytest.mark.require_admin
+
+
+@pytest.mark.e2e
+def test_list_clusters(helper: Helper) -> None:
```

Comment:
These two tests do not require admin. Maybe left them in a different file?

Reply:
Ok, let me mark every test explicitly then.